kernel/packages/decentraland-ecs/types/@decentraland/PortableExperiences/index.d.ts

```diff
@@ -0,0 +1,16 @@
+declare module '@decentraland/PortableExperiences' {
+  type PortableExperienceIdentifier = string
+  type PortableExperienceHandle = {
+    pid: PortableExperienceIdentifier
```

Comment:
should this pid be the sceneUrn instead? I think there is a greater concept here, will use this 👾 emoji to refer to the same topic in the review